Corona-Warn-App/src/main/java/de/rki/coronawarnapp/task/example/QueueingTask.kt

```diff
@@ -0,0 +1,80 @@
+package de.rki.coronawarnapp.task.example
+
+import de.rki.coronawarnapp.task.Task
+import de.rki.coronawarnapp.task.TaskCancelationException
+import de.rki.coronawarnapp.task.TaskFactory
+import de.rki.coronawarnapp.task.common.DefaultProgress
+import kotlinx.coroutines.channels.ConflatedBroadcastChannel
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.asFlow
+import timber.log.Timber
+import java.io.File
+import java.util.UUID
+import javax.inject.Inject
+import javax.inject.Provider
+
+open class QueueingTask @Inject constructor() : Task<DefaultProgress, QueueingTask.Result> {
```

Comment:
Should be removed in the upcomig PRs or moved to the test package and tested

Reply:
Will be moved in an upcoming PR.